docs/installation.md

```diff
@@ -0,0 +1,145 @@
+This document will introduce how to install Arcadia on a Kubernetes cluster.
+
+## Install and Run Arcadia
+
+#### Create or Use existing Kubernetes Cluster
```

Comment:
should this be  `###` ?

Reply:
that's on purpose to make it more smaller